fullscreen.bs

```diff
@@ -109,15 +97,15 @@ its <a>node document</a>'s <a>top layer</a>.
 
   <ol>
    <li><p>If <var>node</var> is its <a>node document</a>'s <a>fullscreen element</a>,
-   <a>exit fullscreen</a> that <a for=/>document</a>.
+   <a>exit fullscreen</a> that <a for=/>document</a> with <var>fullyFlag</var> unset.
 
    <li><p>Otherwise, <a lt="unfullscreen an element">unfullscreen <var>node</var></a> within its
    <a>node document</a>.
   </ol>
 </ol>
 
 <p>Whenever the <a>unloading document cleanup steps</a> run with a <var>document</var>,
-<a>fully exit fullscreen</a> <var>document</var>.
+<a>exit fullscreen</a> <var>document</var> with <var>fullyFlag</var> set.
```

Comment:
I've realized this is actually a bit broken. Exit fullscreen is async and recursive, and
* The async bit means that things should happen to the document after the resize, when it's (probably) already not the active document. In particular, events should be fired, that aren't fired in at least Firefox in https://github.com/w3c/web-platform-tests/pull/5887
* https://html.spec.whatwg.org/multipage/browsers.html#unload-a-document itself recurses into descendant browsing contexts, so there would be more exiting going on than necessary.

I think it would actually be best to just unfullcreen the document with no events at all here, and will change the PR to do that.